SConstruct

```diff
@@ -53,15 +53,18 @@ if env["target"] in ["editor", "template_debug"]:
     except AttributeError:
         print("Not including class reference as we're targeting a pre-4.3 baseline.")
 
-file = "{}{}{}".format(libname, env["suffix"], env["SHLIBSUFFIX"])
+# .dev doesn't inhibit compatibility, so we don't need to key it.
+# .universal just means "compatible with all relevant arches" so we don't need to key it.
+suffix = env['suffix'].replace(".dev", "").replace(".universal", "")
+
+lib_filename = f"{env.subst('$SHLIBPREFIX')}{libname}{suffix}{env.subst('$SHLIBSUFFIX')}"
```

Comment:
Could we maybe leave the `.format` syntax for the strings embedding? I would personally prefer the more explicit syntax.

Reply:
I tried all sorts of syntaxes and wasn't very happy with any of them 😅
If you prefer `.format`, I'm happy to use that!